packages/menu/demo/stories/MenuStory.tsx

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef } from 'react';
+import { StoryFn } from '@storybook/react';
+import classNames from 'classnames';
+import {
+  IMenuContainerProps,
+  IUseMenuProps,
+  IUseMenuReturnValue,
+  IMenuItemBase,
+  MenuItem,
+  MenuContainer,
+  useMenu
+} from '@zendeskgarden/container-menu';
+
+type MenuContainerProps = IMenuContainerProps<HTMLButtonElement, HTMLUListElement>;
+type UseMenuProps = IUseMenuProps<HTMLButtonElement, HTMLUListElement>;
+type MenuReturnValue = IUseMenuReturnValue;
+
+interface IUseMenuComponentProps extends MenuReturnValue {
+  items: UseMenuProps['items'];
+}
+
+type MenuItemProps = {
+  item: IMenuItemBase;
+  getItemProps: IUseMenuComponentProps['getItemProps'];
+  isSelected?: boolean;
+};
+
+const Item = ({ item, getItemProps, isSelected }: MenuItemProps) => (
+  <li
+    className={classNames({
+      'hover:bg-grey-200': !item.disabled,
```

Comment:
I can definitely hook up `focusedValue === item.value` for this background class. 

Also, I believe clicking the menu button then pressing arrow down (or up) would focus the first (or last) menu item since focus isn't in the menu yet. That's the behavior in the [APG examples](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-editor/), anyway.